src/main/java/org/tron/core/db/common/WrappedByteArray.java

```diff
@@ -1,16 +1,30 @@
 package org.tron.core.db.common;
 
 import lombok.Getter;
-import lombok.Value;
-
 import java.util.Arrays;
 
-@Value(staticConstructor = "of")
 public final class WrappedByteArray {
 
   @Getter
   private byte[] bytes;
 
+  public static WrappedByteArray of(byte[] bytes) {
+    return new WrappedByteArray(bytes);
+  }
+
+  public static WrappedByteArray copyOf(byte[] bytes) {
+    byte[] value = null;
+    if (bytes != null) {
+      value = Arrays.copyOf(bytes, bytes.length);
+    }
+
+    return new WrappedByteArray(value);
+  }
+
+  private WrappedByteArray(byte[] bytes) {
+    this.bytes = bytes;

```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Store a copy of "bytes". ([squid:S2384](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2384))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1584&issues=AWZY6VOKrFHbA8-EISln&open=AWZY6VOKrFHbA8-EISln)

[ISSUE_KEY]: AWZY6VOKrFHbA8-EISln